native/pom.xml

```diff
@@ -25,21 +25,19 @@
             <properties>
                 <compilerStartOptionDirectory>-I ${JAVA_HOME}/include/linux/</compilerStartOptionDirectory>
                 <linkerStartOptionDirectory>-I ${JAVA_HOME}/include/linux/</linkerStartOptionDirectory>
-                <packagingDirectory>so</packagingDirectory>
-                <includeDirectory>libiomp5.so</includeDirectory>
-                <typeDirectory>so</typeDirectory>
-                <destFileNameDirectory>libjmkl.so</destFileNameDirectory>
+                <dynamicLibType>so</dynamicLibType>
```

Comment:
Because of the `activeByDefault` tag? So I just need to change `default` to `linux`? @yiheng